src/page-modules/assistant/trip/trip-pattern/index.tsx

```diff
@@ -108,15 +118,27 @@ export default function TripPattern({
                     )}
                   </div>
 
-                  <Typo.span textType="body__tertiary">
-                    {formatToClock(leg.expectedStartTime, language, 'floor')}
-                  </Typo.span>
+                  <div className={style.timeStartContainer}>
+                    <Typo.span textType="body__tertiary">
+                      {formatToClock(leg.expectedStartTime, language, 'floor')}
```

Comment:
Not sure if this matters, to be honest, but I immediately think that we should use `leg.aimedStartTime` here. I get that if they are within the same minute it doesn't matter and the logic would be valid. But hey, just a thought :) It might just be me and this is fine, up to you

Reply:
Should see how this affects the duration of a trip and corresponding start/end times too probably.

Reply:
I assume we want the updated departure to appear first? In that case, I dont think it will display properly if I changed to initially display aimedStartTime instead of expectedStartTime. AimedStartTime is always equal or earlier than expectedStartTime😃

Reply:
It would have to be something in the line of

```tsx 
{ realtime ? 
<p> { leg.expectedStartTime} </p>
<p strike-through> { leg.aimedStartTime } </p>
:
<p> { leg.aimedStartTime } </p>
}
```

As I said, might not matter or be better, but I would think that it would be more clear what times we use and why

Reply:
Does matter more when we now use `secondsBetween()` instead of the `isEqualOnMinuteLevel()`. The former might return true for 13:39:59 and 13:40:58, but are two different minutes.

Reply:
Good point

Reply:
True, added it back now.